akka-projection-core/src/main/scala/akka/projection/scaladsl/ProjectionManagement.scala

```diff
@@ -100,6 +106,24 @@ class ProjectionManagement(system: ActorSystem[_]) extends Extension {
     retry(() => askSetOffset())
   }
 
+  /**
+   * Update the stored `TimestampOffset` for the `projectionId` and restart the `Projection`.
+   * This can be useful if the projection was stuck with errors on a specific offset and should skip
+   * that offset and continue with next.
+   *
+   * Another use case is to populate the offset store with know starting points when enabling change events
+   * for Durable State. In that case the offset sequence number should be set to current Durable State
+   * revision minus 1.
+   */
+  def updateTimestampOffset(projectionId: ProjectionId, updates: Set[UpdateTimestampOffset]): Future[Done] = {
+    def askSetTimestampOffset(): Future[Done] = {
+      topic(projectionId.name)
+        .ask(replyTo => Topic.Publish(SetTimestampOffset(projectionId, updates, replyTo)))
+    }
+
+    retry(() => askSetTimestampOffset())
+  }
```

Comment:
How would one handling starting a new projection from the offsets, hold back the consumer until the response comes back from this API, because it would start consuming from start as soon as it has started executing, right?